### ISSUE TITLE
"Element query executed before Craft is fully initialized" warning when using craft-sentry

### ISSUE THREAD

**Issue by BigglesZX** [born05/craft-sentry#21]
Hi all. I'm getting the following warning in the logs panel of the Craft debug toolbar and I think I've narrowed it down to this plugin. When the plugin is enabled I get:

```
craft\elements\db\ElementQuery::prepare
Element query executed before Craft is fully initialized.
```

Project stack:

* PHP 8.1.7
* MySQL 8.0.27
* Craft Solo 4.1.2
* craft-sentry 2.0.0-beta.1

My config file:

```php
<?php

return [
    '*' => [
        'enabled'        => true,
        'anonymous'      => false,
        'clientDsn'      => getenv('SENTRY_DSN'),
        'excludedCodes'  => ['400', '404', '429'],
        'release'        => getenv('SENTRY_RELEASE') ?: null,
        'reportJsErrors' => false,
        'sampleRate'     => 0,
    ],
];
```

Note that I've also tried `'anonymous' => true` but that doesn't seem to make any difference.

Let me know if I can provide any other info. Thanks!

**Comment by roelvanhintum** [comment]
Thanks @BigglesZX! This is fixed in 2.0.0

**Comment by BigglesZX** [comment]
Thanks!
